README.ja.rst

```diff
@@ -468,14 +468,11 @@ pikzie.priority(priority)
 謝辞
 ----
 
-  * aztmさん
-
-    * バグレポートをしてくれました。
-
-    * ebuildを作ってくれました。
-      http://diary.atzm.org/20081201.html#p01
-
-  * Hideo Hattoriさん
+- aztmさん
+ 	バグレポートをしてくれました。
```

Comment:
Why do you change to definition list from nested itemization?
It seems that this change breaks the original meaning.

The original: https://github.com/clear-code/pikzie/blob/master/README.ja.rst#%E8%AC%9D%E8%BE%9E
The changed: https://github.com/shopetan/pikzie/blob/fixreadme/README.ja.rst#%E8%AC%9D%E8%BE%9E

The original means that "aztm" (oh, it's a typo... He is "atzm" not "aztm"...) has two contributions. But the changed means that "aztm" has only one contribution.